tests/debugger_visualizer.rs

```diff
@@ -0,0 +1,44 @@
+use debugger_test::debugger_test;
+use indexmap::IndexMap;
+use indexmap::IndexSet;
+
+#[inline(never)]
+fn __break() {}
+
+#[debugger_test(
+    debugger = "cdb",
+    commands = r#"
+.nvlist
+dx set
+dx map
+"#,
+    expected_statements = r#"
+set              : { len=0x63 } [Type: indexmap::set::IndexSet<i32,std::collections::hash::map::RandomState>]
+[len]            : 0x63 [Type: unsigned __int64]
+[0]              : 0 [Type: indexmap::Bucket<i32,tuple$<> >]
+[1]              : 1 [Type: indexmap::Bucket<i32,tuple$<> >]
+[98]             : 98 [Type: indexmap::Bucket<i32,tuple$<> >]
+
+map              : { len=0x63 } [Type: indexmap::map::IndexMap<i32,i32,std::collections::hash::map::RandomState>]
+[len]            : 0x63 [Type: unsigned __int64]
+[0]              : { key=0, value=0 } [Type: indexmap::Bucket<i32,i32>]
+[1]              : { key=1, value=-1 } [Type: indexmap::Bucket<i32,i32>]
+[2]              : { key=2, value=-2 } [Type: indexmap::Bucket<i32,i32>]
+[98]             : { key=98, value=-98 } [Type: indexmap::Bucket<i32,i32>]
```

Comment:
Is the `[N]` part showing the positional index? I expect so, but it's confusing that you used the same key values.

Reply:
`[N]` is the positional index of each `Bucket` in the underlying `Vec` structure. I can update the examples to ease the confusion.

Reply:
Thanks, the new examples are clear, and more fun! :slightly_smiling_face: